modules/web/css/ballerina-file-editor.css

```diff
@@ -892,7 +892,7 @@ input#package-name-input {
 }
 
 .annotation-table td{
-    height: 20px;
+    height: 18px;
```

Comment:
Having height 18px might cause an issue with calculating the height of the container as the default height is taken as 20px. Have to update here as well https://github.com/ballerinalang/composer/blob/master/modules/web/js/ballerina/visitors/sizing-utils.js#L595 .